### ISSUE TITLE
Image Elements

### ISSUE THREAD

**Issue by alfwall** [alfwall/cbc-mod1c-Horiseon#3]
The first image is hidden within a div whose class conceals the image info, and all 6 other images are missing alt text.

**Comment by alfwall** [comment]
Finished via #7
